update.mjs

```diff
@@ -145,6 +147,7 @@ const readme = [
   'contributors',
   'maintainers',
   'credits',
+  'readme*.md'
```

Comment:
This has already been covered by the first item

Reply:
Yes, I also just found out after pull request